Greetings.go

```diff
@@ -0,0 +1,13 @@
+package main
+
+import "fmt"
+
+func main() {
+
+    names := []string{"Yani", "Vlad", "Krasi", "Ivan", "Kris", "Naso", "Dimi", "Geri", "Lili", "Zoya"}
+
+    for i, c := range names {
+        fmt.Println("", i)
+        fmt.Println("", c)

```

Comment:
Why do you have an empty string `""` ?

Reply:
Fix it :)

Reply:
I don't see the changes, need to push the new  changes(new commit) to the "Greetings" branch.

Reply:
Do you see it now?